commit 454da5e7b24c3e8bc4b0e61ca4460fdcc90d5168
Author: Kefu Chai <kchai@redhat.com>
Date:   Fri Feb 24 20:38:03 2017 +0800

    mon: pass const variables by const ref not pointer
    
    * PGMapUpdater::check_down_pgs(): pass a const reference to pgmap
      instead of a pointer
    * PGMapUpdater::register_new_pgs(): pass a const reference to pgmap
      instead of a pointer
    
    Signed-off-by: Kefu Chai <kchai@redhat.com>

diff --git a/src/mgr/ClusterState.cc b/src/mgr/ClusterState.cc
index c689eb310a7..0e224adadcb 100644
--- a/src/mgr/ClusterState.cc
+++ b/src/mgr/ClusterState.cc
@@ -100,13 +100,13 @@ void ClusterState::notify_osdmap(const OSDMap &osd_map)
   PGMap::Incremental pending_inc;
   pending_inc.version = pg_map.version + 1; // to make apply_incremental happy
 
-  PGMapUpdater::update_creating_pgs(osd_map, &pg_map, &pending_inc);
-  PGMapUpdater::register_new_pgs(osd_map, &pg_map, &pending_inc);
+  PGMapUpdater::update_creating_pgs(osd_map, pg_map, &pending_inc);
+  PGMapUpdater::register_new_pgs(osd_map, pg_map, &pending_inc);
 
   // brute force this for now (don't bother being clever by only
   // checking osds that went up/down)
   set<int> need_check_down_pg_osds;
-  PGMapUpdater::check_down_pgs(osd_map, &pg_map, true,
+  PGMapUpdater::check_down_pgs(osd_map, pg_map, true,
 			       need_check_down_pg_osds, &pending_inc);
 
   pg_map.apply_incremental(g_ceph_context, pending_inc);
